Source/Shaders/Builtin/Functions/planeDistance.glsl

```diff
@@ -11,3 +11,18 @@
 float czm_planeDistance(vec4 plane, vec3 point) {
     return (dot(plane.xyz, point) + plane.w);
 }
+
+/**
+ * Computes distance from an point to a plane, typically in eye space.
```

Comment:
"a point"

Also, "typically in eye space" is not needed in this context right.  All the function parameters just need to be in the coordinate system.